app/graphql/schema.graphql

```diff
@@ -323,18 +323,7 @@ type OrdinalMeasure implements Dimension {
   hierarchy(sourceType: String!, sourceUrl: String!): [HierarchyValue!]
 }
 
-scalar SearchCubeCreator
-scalar SearchCubeThemes
-
-type SearchCube {
-  iri: String!
-  title: String!
-  description: String
-  creator: SearchCubeCreator
-  publicationStatus: DataCubePublicationStatus!
-  datePublished: String
-  themes: SearchCubeThemes!
-}
+scalar SearchCube
```

Comment:
I don't quite understand why it's good to use a scalar vs a type here ? What is the difference ?

Reply:
Scalar was used to stop resolvers chain, as we already have all the data we need at this level, and could avoid additional queries by telling Apollo we want to stop right here :)

Reply:
Yes perfect, I did not have the "resolver chain stopping" in my mind. Super nice perf improvement.